src/config/serverConfigDefaults.ts

```diff
@@ -47,6 +47,7 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     show_mutation_mapper_tool_grch38: true,
     show_transcript_dropdown: false,
     show_signal: true,
+    show_ndex: true,
```

Comment:
@inodb should this really default to true?

Reply:
thanks for checking - yeah i think it makes sense to have this be the default for all installations